### DISCUSSION TITLE
How to configure streamers?

### DISCUSSION THREAD

**Discussion by nexus667** [RakambdaOrg/ChannelPointsMiner#451] [Q&A] [answered]
I'm running a config with multiple accounts and was wondering how to setup streamers in the main config file.  I got it working somewhat by creating seperate streamer files in the streamers folder, but it does not like the parameters in my streamer config and uses the default settings  which I set in my main config.  So from that I have 2 questions:

1.  How to configure streamers in main config?
2. What am I doing wrong in the streamer's config file?

ex. Streamer config:

```json
(
	"claimMoments" : true,
	"joinChat" : true, 
        "participateCampaigns" : true
)
```

ex.  Main config:

```json
{
  "accounts": [
    {
      "username" : "mt7",
      "loginMethod" : {
      "type" : "tv",
      "password" : ""
      },
      "defaultStreamerSettings" : {
        "claimMoments" : true,
        "joinChat" : true,
        "participateCampaigns" : true
      },
      "streamerConfigDirectories" : [
        {
          "path" : "streamers",
          "recursive" : true
        }
      ]
    },
...  ## ditto for Additional accounts
```

**Comment by Rakambda** [comment, marked as answer]
```json
(
...
)
```
Parenthesis are not a valid JSON syntax. Use curly brackets.

**Reply by nexus667** [reply]
This answers question #2.  What about configuring streamers in the main config file?

**Reply by Rakambda** [reply]
You can define default settings in `defaultStreamerSettings` and they'll be applied to all streamers unless overriden.

To add streamers, either you activate the `loadFollows` option and the channels you follow will be loaded with the default settings. Or you add a file named `<streamer>.json` within the configured `streamerConfigDirectories`.
If you you imagined is to have a list of streamers within the global config, it is not possible, this is what the folder/file structure does.
